api/src/main/java/org/hawkular/apm/api/model/events/SourceInfo.java

```diff
@@ -27,18 +27,18 @@
 import org.hawkular.apm.api.utils.SerializationUtil;
 
 /**
- * This class represents information cached about a producer, to enable it to be
+ * This class represents information cached about a source node, to enable it to be
  * correlated to a consumer.
  *
  * @author gbrown
  */
-public class ProducerInfo implements Externalizable, ApmEvent {
```

Comment:
This was removed when I getId() was changed to return a list of ids - but later had to revert it. So I can add it back,

However it wasn't a problem as ProducerInfo was never used as an ApmEvent (which I think is used for storage) - but I'll add back anyway.